cre.cpp

```diff
@@ -2591,6 +2640,10 @@ static const struct luaL_Reg credocument_meth[] = {
 	{"getHeaderHeight", getHeaderHeight},
 	{"getToc", getTableOfContent},
 	{"getVisiblePageCount", getVisiblePageCount},
+    {"getNextVisibleWordStart", getNextVisibleWordStart},
+    {"getNextVisibleWordEnd", getNextVisibleWordEnd},
```

Comment:
Ok, looks like we have tabs here :| Best to adjust to your surroundings and use a tab too.

Reply:
Or update the whole thing :-p

Reply:
Well, that's strange, I'm using Ubuntu on VM with tab set as 4 spaces, I've probably copied this part from somewhere else. I've checked, and the whole thing uses tabs instead of spaces. I've updated the parts I've noticed(2 arrays where lua functions are defined).

Reply:
Allright :)

Reply:
Heathens! tabs 4 evah'!

:dagger:

/me goes back into his beautiful land of 8-characters wide tabs ;p.

Reply:
I prefer tabs too but the house style is four spaces because, um, it is. xD